### ISSUE TITLE
`@stylistic/jsx-closing-tag-location`: Broken in ternary expression

### ISSUE THREAD

**Issue by ej612** [eslint-stylistic/eslint-stylistic#440]
### Validations

- [X] Follow our [Code of Conduct](https://github.com/eslint-stylistic/eslint-stylistic/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://eslint.style/contribute/guide).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.

### Describe the bug

I have the following code:
```tsx
const component = (true
    ? <div>
        <div/>
    </div>
    : null
);
```

Which shows the following error:
![image](https://github.com/eslint-stylistic/eslint-stylistic/assets/110408767/4a327b30-6806-4f11-b5fa-51d1c1a43e36)

If I follow that suggestion, I now get an error from `jsx-indent`:
![image](https://github.com/eslint-stylistic/eslint-stylistic/assets/110408767/5343f7d8-cd94-44cd-81dd-079ae36419e4)

The expected indentation is the one proposed by `jsx-indent`, it therefore looks as if `jsx-closing-tag-location` doesn't work well in this context.
This is pretty much the same bug as https://github.com/eslint-stylistic/eslint-stylistic/issues/439, except with a different rule.

### Reproduction

N/A

### Contributes

- [ ] I am willing to submit a PR to fix this issue
- [ ] I am willing to submit a PR with failing tests

**Comment by 9romise** [comment]
Closing, as described in #378 and #413, you can just use `indent` instead of `jsx-indent`. It passes all the `jsx-indent` test cases.
And there is an option(`offsetTernaryExpressions`) in `indent` to control the `TernaryExpressions`.
